docs/intro/quickstart/working.rst

```diff
@@ -0,0 +1,412 @@
+.. _ref_quickstart_working:
+
+=====================
+Working with our data
+=====================
+
+.. edb:split-section::
+
+  With TypeScript, there are three ways to run a query: use a string EdgeQL query, use our ``queries`` generator to turn a string of EdgeQL into a TypeScript function, or use our query builder API to build queries dynamically in a type-safe manner. In the next example, we'll show you each of these methods, but for the rest of the tutorial we'll use the query builder API.
```

Comment:
```suggestion
  With TypeScript, there are three ways to run a query: use a string EdgeQL query, use our ``queries`` generator to turn a string of EdgeQL into a TypeScript function, or use our query builder API to build queries dynamically in a type-safe manner. In the next example, we'll show you each of these methods, but for the rest of this section of the tutorial we'll use the query builder API.
```

?

Reply:
I use the query builder for the entire tutorial, there are no examples of using the other two methods. Should there be?